deployers/servicedeployer.go

```diff
@@ -1136,9 +1137,15 @@ func (deployer *ServiceDeployer) UnDeployDependencies() error {
 
 func (deployer *ServiceDeployer) UnDeployPackages(deployment *DeploymentProject) error {
 	for _, pack := range deployment.Packages {
-		err := deployer.deletePackage(pack.Package)
-		if err != nil {
-			return err
+		// "default" package is a reserved package name
+		// all openwhisk entities were deployed under
+		// /<namespace> instead of /<namespace>/<package> and
+		// therefore skip deleting default package during undeployment
+		if strings.ToLower(pack.Package.Name) != parsers.DEFAULT_PACKAGE {
```

Comment:
Do we still need to delete it from /namespace ?

Reply:
What about update/sync?

Reply:
Nope, we do not have to delete it from /namespace as its a reserved package. Its can neither be created nor be deleted. Its pretty much same as having no package and therefore the entities are created directly under `/namespace` except in case of web actions where action URL has `default` specified in it.